sparse_ho/criterion.py

```diff
@@ -79,75 +69,76 @@ def compute_rmse(self, mask, dense, beta_star):
         else:
             self.rmse = None
 
-    def get_val(self, log_alpha, tol=1e-3):
+    def get_val(self, model, X, y, log_alpha, tol=1e-3):
```

Comment:
I don't get the distinction between this and get_mse_val. What does it do ?

Reply:
The ```get_val```  function is for 0 order méthods, it compute the coefficients beta (without Jacobians) and the value of the criterion, whereas  ```get_mse_val``` compute the mse given the coefficients beta.

Reply:
for me it's name shouldn't be get_val if it does something else. what do you think ?

Reply:
also, I got confused between MSE in prediction (the name of the class) and MSE in estimation. I would rename get_mse to get_mse_estim ?

Anyway, this is not something that the end used wants (he does not have beta_star). Couldn't it be handled with a callback ?

Reply:
> Anyway, this is not something that the end used wants (he does not have beta_star). Couldn't it be handled with a callback ?

ye it will handle in a callback